package/gluon-ssid-changer/check_site.lua

```diff
@@ -0,0 +1,9 @@
+need_boolean({'ssid_changer', 'enabled'}, false)
+need_number({'ssid_changer', 'switch_timeframe'}, false)
+need_number({'ssid_changer', 'first'}, false)
+need_string({'ssid_changer', 'prefix'}, false)
+need_one_of({'ssid_changer', 'suffix'}, {'nodename', 'mac', 'none'}, false)
+if need_boolean({'ssid_changer','tq_limit_enabled'}, false) then
```

Comment:
In case we switch to required site.conf variables, the `tq_limit_enabled` could be dropped and implicitly assumed by a dictionary containing `tq_limit_max` and `tq_limit_min` (see example below)

```
ssid_changer = {
  ..
  tq_limit = {
    max = xx,
    min = xx,
  },
  ..
},
```

Reply:
i'd rather like to have `tq_limit_enabled` as required and `tq_limit_min` and max optional with sound default values